### ISSUE TITLE
Custom stop intent

### ISSUE THREAD

**Issue by ahmadabdullah247** [actions-on-google/actions-on-google-nodejs#292]
So I am trying to add Stop intent in my action. When i test it on my google home device, it pauses the whole action and never calls my intent. Is it the default behavior for google home? is Stop a keyword in google home? If yes can we override it?

**Comment by Fleker** [comment]
There are multiple ways a user can always exit an action. "Stop" is one, and there are several others. These are global, so you can't prevent it from happening. What you can do is override the response and add a short good-bye message.

See the documentation on conversation exits: https://developers.google.com/actions/assistant/conversation-exits

**Comment by ahmadabdullah247** [comment]
@Fleker I think I didn't explained my question well enough, Let me rephrase. I have two different intents.
1. exit intent : which closes my action
2. stop intent : which should say a sentence. 
But in my implementation when I say stop, in the simulator it exits (where it should say the sentence from stop intent). On google home device it just pauses the execution of action.

**Comment by Fleker** [comment]
Saying "stop" is defined globally as an exit phrase, so exiting the action is intentional, and there's no way for it to not exit.

**Comment by igilham** [comment]
True, but bear in mind that there are various bugs in Assistant around terminating the interaction. Saying "cancel" or "stop" does not always work correctly on all surfaces.
